nigsp/tests/test_statistics.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+from numpy.random import rand
+from pytest import raises
+
+from nigsp.operations.statistics import two_level_statistical_model
+
+
+# ### Unit tests
+def test_two_level_statistical_model():
+    # Set up zero everywhere except specific roi, aka = signal on the other rois are equivalent to noise
+    # Tests specifically at the rois having signal
+    n_events = 30
+    n_surrogates = 40
+    n_roi = 360
+    n_subs = 20
+    empi_data = np.zeros((n_events, n_subs, n_roi))
+    surr_data = np.zeros((n_events, n_surrogates, n_subs, n_roi))
+
+    rois = np.random.choice(360, size=10, replace=False)  # no replacement
+
+    for roi in rois:
+        random_empi = np.random.rand(n_events * n_subs)
+        random_surr = np.random.rand(n_events * n_surrogates * n_subs)
+        empi_data[:, :, roi] = random_empi.reshape(n_events, n_subs)
+        surr_data[:, :, :, roi] = random_surr.reshape(n_events, n_surrogates, n_subs)
+    n_perms = 50
+    second_level_stats = two_level_statistical_model(
+        empi_data,
+        surr_data,
+        n_perms=n_perms,
+        output_dir_first_level="/tmp",
+        output_dir_second_level="/tmp",
+    )
+
+    assert second_level_stats.shape == (n_roi,)  # do we get tstats for each roi?
+    assert (
+        second_level_stats[rois] != 0
+    ).all()  # do we get tstats for predefined rois?
+    assert np.isnan(second_level_stats).sum() == n_roi - len(
+        rois
+    )  # do we get nan for the rest of the rois?
+
+    # Stability test
+    second_level_stats_repeat = two_level_statistical_model(
+        empi_data, surr_data, n_perms=n_perms
+    )
+    assert np.isclose(second_level_stats[rois], second_level_stats_repeat[rois]).all()
+
+
+### Break tests
+def break_test_two_level_statistical_model():
```

Comment:
```suggestion
def test_break_two_level_statistical_model():
```

Otherwise, pytest won't run this test ;)

Reply:
Lol; congrats and thanks, you just solved one of my weekend mysteries!! break_test and test_break too indiscernible on a Sunday afternoon :D